### PR TITLE
ci!: enforce flake8 through Travis

### PR DESCRIPTION
**Pull request by redeboer**
Part 4 of #78

### CODE REVIEW

File: examples/python/workflow.py [hunk 1]
```diff
@@ -66,12 +69,16 @@
...
-    for name, value in estimator.parameters.items():
+    for name in estimator.parameters:
         if name in free_params:
             params[name] = free_params[name]
     logging.info(params)
 
     logging.info("starting fit")
-    minu2 = Minuit2()
-    result = minu2.optimize(estimator, params)
+    minuit2 = Minuit2()
```
Comment (redeboer) [inline examples/python/workflow.py:78]: Sorry this isn't a flake error, but popped up in cSpell, apparently

File: tox.ini [hunk 2]
```diff
@@ -25,13 +26,23 @@ exclude =
 ignore = # more info: https://www.flake8rules.com/
     D102 # method docstring
     D103 # function docstring
+    D105 # docstring in magic method
     D107 # init docstring
+    D400 # method docstring
```
Comment (spflueger) [inline tox.ini:31]: This is first line ending with a period or? Why that has to be ignored?

Comment (redeboer) [reply tox.ini:31]: Actually I think we can get rid of the D warnings under flake by just removing flake8-docstring. Let me give that a try

Comment (redeboer) [reply tox.ini]: `flake8-docstrings` indeed causes these warnings and `pydocstyle` makes them redundant 45e473d9932998ec0b965a3257713db2c6f769b7

**Pull request merged**